apps/bestofjs-nextjs/src/lib/global.d.ts

```diff
@@ -104,13 +104,19 @@ declare namespace BestOfJS {
     projects: BestOfJS.Project[];
   }
 
-  type HallOfFameMember = {
-    username: string;
+  interface RawHallOfFameMember {
```

Comment:
Raw means API's data model?

Reply:
> 

The `Raw*` objects are the ones we get from the JSON files (the "static API" that is rebuilt everyday)